@here/harp-mapview/lib/MapView.ts

```diff
@@ -1273,6 +1309,17 @@ export class MapView extends THREE.EventDispatcher {
         this.update();
     }
 
+    /**
+     * [[UriResolver]] used to resolve application/deployment specific `URI`s into actual `URLs`
+     * that can be loaded with `fetch`.
+     */
+    get uriResolver(): UriResolver | undefined {
+        return this.m_uriResolver;
+    }
+    set uriResolver(uriResolver: UriResolver | undefined) {
+        this.m_uriResolver = uriResolver;
```

Comment:
Not sure if it's a good idea to add this setter. To make this work properly the theme would need to be re-resolved.